osv/ecosystems.py

```diff
@@ -349,10 +355,19 @@ def enumerate_versions(self, package, introduced, fixed, limits=None):
       ) from ex
 
     response = json.loads(text_response)
-    versions: list[str] = [x['version'] for x in response['result']]
+    raw_versions: list[str] = [x['version'] for x in response['result']]
+
+    # Remove rare cases of unknown versions
+    def version_is_valid_log(v):
```

Comment:
nit: no need for `_log`. It's a bit confusing what the return value means otherwise.